src/App.vue

```diff
@@ -1,10 +1,26 @@
 <script>
+const sidebarSize = { max: 600, min: 350 }
+
 import Connections from '@/components/connections.vue'
 
 export default {
   name: 'App',
+
   components: {
     Connections
+  },
+
+  methods: {
+    mouseDown(e) {
```

Comment:
at this point do you think it would make sense to extract sidebar into its own component?